src/Logging/messages.lua

```diff
@@ -10,6 +10,7 @@ return {
 	cannotConnectEvent = "The %s class doesn't have an event called '%s'.",
 	cannotCreateClass = "Can't create a new instance of class '%s'.",
 	computedCallbackError = "Computed callback error: ERROR_MESSAGE",
+	invalidPropertyType = "The class type '%s' has property '%s', but you attempted to set it to a '%s', while it expected a '%s'.",
```

Comment:
This is a bit awkwardly worded - may I suggest something along the lines of:

> 'Position' expected a 'UDim2', but got a 'nil' (for class 'Frame')

Reply:
Thanks for the feedback! I agree that this is awkwardly worded. I'll work on addressing that right now.

Reply:
I pushed a fix for this issue, but I'm leaving this comment unresolved so you can verify this fix properly address your comment.